core/themes/seven/templates/maintenance-page.tpl.php

```diff
@@ -27,9 +27,5 @@
     </main>
   </div>
 
-  <footer role="contentinfo">
-    <?php print $footer; ?>
-  </footer>
-
```

Comment:
yeah sorry that commit was on my 1.x branch for some reason. Should be gone now :)